tests/main.js

```diff
@@ -60,11 +60,17 @@ export async function test(cb = (_mockPool) => {}, env = DEFAULT_ENV) {
   const owner = env.INPUT_OWNER ?? env.GITHUB_REPOSITORY_OWNER;
   const currentRepoName = env.GITHUB_REPOSITORY.split("/")[1];
   const repo = encodeURIComponent(
-    (env.INPUT_REPOSITORIES ?? currentRepoName).split(",")[0]
+    (env.INPUT_REPOSITORIES ?? currentRepoName).split(",")[0],
   );
+
+  const getInstallationPath = `${basePath}/repos/${owner}/${repo}/installation`;
+
   mockPool
     .intercept({
-      path: `${basePath}/repos/${owner}/${repo}/installation`,
+      path(pathString) {
+        console.log(`\nGET ${pathString}\n`);
```

Comment:
@parkerbxyz we didn't verify that the expected requests are sent to GitHub, we just mocked them. I added two `console.log` statements to both mocks in `main.js` and updated the snapshots